### ISSUE TITLE
Enabled loading library in Eclipse

### ISSUE THREAD

**Issue by devilmac** [timroes/EnhancedListView#51]
Hi timroes,

I was able to add your library in my Eclipse project by doing the following:
- in build.gradle file inside the library folder (not the root folder project), I added this line: **apply plugin: 'eclipse'**;
- I had to add the ANDROID_HOME environment variable pointing to the root of Android SDK (it needs to reboot the system before it works; if this wouldn't work, add to the root folder project a **local.properties** file containing this line: **sdk.dir=pathToAndroidSDK**);
- in a terminal, I have executed this command from the root folder of the project:
  **gradlew eclipse**.

When the process is finished, inside the library folder there were these three files:
- **.classpath** file;
- **.project** file;
- **.settings** folder.

In other words, the command above has "translated" the gradle project into an Eclipse project.

Done that, I have imported the project into my Eclipse workspace, made changes to get a valid project without any error, set it to be an Android library and added it to my application as usual in Eclipse.

I hope this will help.

**Comment by timroes** [comment]
Hei, might I add this to the wiki for explanation?

**Comment by devilmac** [comment]
Hi,

YES! OF COURSE! We are here to help other programmers, isn't it? ;)
I've retried the procedure again a few moments ago, just to be safe, and it works.
After this "translation" it needs to add "nineoldandroids" library to the Eclipse project. Other adjustments are to move the source folder in the right location to make Eclipse's project error free.

EDIT

I'm sorry if I have open an issue for this. Last but not least, if you could mention me as the author of this procedure on your wiki, I would be grateful :)
